packages/engine/Source/Core/Resource.js

```diff
@@ -2241,7 +2203,7 @@ Resource._Implementations.loadWithXhr = function (
     } else if (
       (browserResponseType === "" || browserResponseType === "document") &&
       defined(xhr.responseXML) &&
-      xhr.responseXML.hasChildNodes()
+      xhr.responseXML?.hasChildNodes()
```

Comment:
With optional chaining here, the previous `defined` check is redundant.

Reply:
If I drop the `defined` check that's there it would be relying on the last line to be falsey when it's `undefined`. Normally that would mean we should wrap it in a `defined` check for consistency but that doesn't equate to the same check. Not sure what the pattern/style normally is for CesiumJS, thoughts?

cc @ggetz